src/Widgets/DataFormBooleanWidget/DataFormBooleanWidgetEditingConfig.ts

```diff
@@ -10,21 +10,47 @@ provideEditingConfig(DataFormBooleanWidget, {
       title: 'Name of the data attribute in question',
     },
     required: { title: 'Mandatory?' },
+    dataForNo: {
+      title: 'Attribute value for “no” (optional)',
+      description: 'Default: boolean false',
+    },
+    dataForYes: {
+      title: 'Attribute value for “yes” (optional)',
+      description: 'Default: boolean true',
+    },
     defaultValue: {
       title: 'Default value',
       description:
         'Only relevant for a create form. An update form will utilize the existing boolean value.',
     },
     helpText: { title: 'Help text' },
+    style: {
+      title: 'Style',
+      values: [
+        { value: 'check', title: 'Checkbox' },
+        { value: 'switch', title: 'Switch' },
+      ],
+    },
+    submitOnChange: { title: 'Submit on change?' },
```

Comment:
How about the title `'Immediately submit changed value?'`? Also a description should be added, that this will not apply for new items, only for updating existing items.